modules/RTC/TraceablePeerConnection.js

```diff
@@ -2911,16 +2914,15 @@ TraceablePeerConnection.prototype._processLocalSSRCsMap = function(ssrcMap) {
 
             // eslint-disable-next-line no-negated-condition
             if (newSSRCNum !== oldSSRCNum) {
-                oldSSRCNum && logger.error(`${this} Overwriting SSRC for [track=${track},id=${trackMSID}]`
-                    + ` with ssrc=${newSSRC}`);
+                oldSSRCNum && logger.error(`${this} Overwriting SSRC for track=${track}] with ssrc=${newSSRC}`);
                 this.localSSRCs.set(track.rtcId, newSSRC);
                 this.eventEmitter.emit(RTCEvents.LOCAL_TRACK_SSRC_UPDATED, track, newSSRCNum);
             }
         } else if (!track.isVideoTrack() && !track.isMuted()) {
             // It is normal to find no SSRCs for a muted video track in
             // the local SDP as the recv-only SSRC is no longer munged in.
             // So log the warning only if it's not a muted video track.
-            logger.warn(`${this} No SSRCs found in the local SDP for track[track=${track},id=${trackMSID}]`);
+            logger.warn(`${this} No SSRCs found in the local SDP for track=${track}`);
```

Comment:
can you also include the `sourceIdentifier` in the log? just to know what id was not found